anyway/parsers/scrape_arabic_news.py

```diff
@@ -0,0 +1,125 @@
+
+import requests
+import re
+import json
+from datetime import datetime
+
+
+class ArticleMetaData:
+    article_pub_date: datetime
+    abstract: str
+    title: str
+
+    def __init__(self, _article_pub_date, _abstract, _title):
+        self.article_pub_date = _article_pub_date
+        self.abstract = _abstract
+        self.title = _title
+
+
+def scrape(panet_page):
+
+    website_content = requests.get(f"http://www.panet.co.il/category/home/2/{panet_page}")
+
+    """ Historic article titles: """
+
+    regex1 = r"<div class=\"panet-title\">(.*?)<\/div>"
+    matches1 = re.finditer(regex1, website_content.text, re.MULTILINE)
+
+    with open(f"titles_{panet_page}.txt", "w", encoding="utf-8") as f1:
+
+        for matchNum, match in enumerate(matches1, start=1):
+
+            for groupNum in range(0, len(match.groups())):
+                groupNum = groupNum + 1
+                f1.write(match.group(groupNum))
```

Comment:
Hi,
- In python we use lowercase and underscore for variables and function/method names.
- I prefer not to change the variable loop in the loop. It is better to use a different variable.